addons/common/fnc_isTurnedOut.sqf

```diff
@@ -27,54 +27,53 @@ Examples:
 
 Author:
     Nou, courtesy of ACRE project.
+    shukari
 ---------------------------------------------------------------------------- */
 SCRIPT(isTurnedOut);
 
 params ["_unit"];
 
-private _turn = false;
-private _out = false;
+if (isNull objectParent _unit) exitWith {true};
+if (isTurnedOut _unit) exitWith {true};
+
+private _return = false;
 private _vehicle = vehicle _unit;
-private _cfg = configFile >> "CfgVehicles" >> (typeOf _vehicle);
-
-if (_vehicle != _unit) then {
-    if (driver _vehicle == _unit) then {
-        private _forceHideDriver = getNumber(_cfg >> "forceHideDriver");
-        _turn = (_forceHideDriver != 1);
-        _out = (_vehicle animationPhase "hatchDriver") > 0;
-    } else {
-        private _assignedRole = assignedVehicleRole _unit;
-        if (_assignedRole select 0 == "Turret") then {
-            private _turretPath = _assignedRole select 1;
-            private _turret = [_vehicle, _turretPath] call CBA_fnc_getTurret;
-
-            private _canHideGunner = getNumber(_turret >> "canHideGunner");
-            private _forceHideGunner = getNumber(_turret >> "forceHideGunner");
-            _turn = (_canHideGunner == 1 && _forceHideGunner == 0);
-
-            private _hatchAnimation = getText(_turret >> "animationSourceHatch");
-            _out = (_vehicle animationPhase _hatchAnimation) > 0;
-
-        } else {
-            if ((_assignedRole select 0) == "Cargo") then {
-                private _attenuateCargo = getArray(_cfg >> "soundAttenuationCargo");
-                if ((count _attenuateCargo) > 0) then {
-                    private _index = (count _attenuateCargo)-1; // wait for command to get cargo index
-
-                    if (_index > -1) then {
-                        if(_index > (count _attenuateCargo)-1) then {
-                            _index = (count _attenuateCargo)-1;
-                        };
-                        if((_attenuateCargo select _index) == 0) then {
-                            _out = true;
-                        };
-                    };
-                };
-            };
-        };
+private _role = _unit call CBA_fnc_vehicleRole;
+
+// FFVs are allways outside
+if ([_unit] call ace_common_fnc_getTurretIndex in ([_vehicle] call ace_common_fnc_getTurretsFFV)) then {_return = true};
+
+// Spezial condition for RHS car windows
+if (_vehicle isKindOf "rhsusf_m1025_w" && {_role in ["driver", "Turret"]}) then {
+    private _cargoIndex = (_vehicle getCargoIndex _unit) min 2;
+    private _window = ['ani_window_1', 'ani_window_2', 'ani_window_4', 'ani_window_3'] select (_cargoIndex + 1);
+    _return = (_vehicle doorPhase _window) > 0;
+};
+if (_return) exitWith {_return}; // return
```

Comment:
kinda useless comment

Reply:
no, else the return true will be over riden by the exitwith conditions